base/compiler/stmtinfo.jl

```diff
@@ -114,6 +114,7 @@ Each (abstract) call to `iterate`, corresponds to one entry in `ainfo.each::Vect
 """
 struct AbstractIterationInfo
     each::Vector{CallMeta}
+    complete::Bool
```

Comment:
Does this change is necessary for this PR or it is a separate fix on the inlining algorithm?

Reply:
This PR adds all the calls into the `.each` field (partly to look at the affects, but mostly for Cthulhu's benefit). Previously the inlining algorithm used the presence or absence of this info to determine whether or not we had the complete set of iteration items. Now we need to keep track of it separately.